plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/GradleEnclavePlugin.kt

```diff
@@ -506,8 +506,8 @@ class GradleEnclavePlugin @Inject constructor(private val layout: ProjectLayout)
             val enclaveBundleJarTask = target.createTask<Jar>("enclaveBundle${type}Jar") { task ->
                 task.group = CONCLAVE_GROUP
                 task.description = "Compile an ${type}-mode enclave that can be loaded by SGX."
-                task.archiveBaseName.set("enclave-bundle")
-                task.archiveAppendix.set(typeLowerCase)
+                task.archiveAppendix.set("-bundle")
```

Comment:
The dash is added by the Jar task already. Also, I prefer "enclave-bundle" as it's a bit more clear if someone looks at the file name.

Reply:
Will remove the dash, you are right. But we I would avoid "enclave", it will mostly always be duplicate.
we would have `enclave-enclave-bundle-simulation`, or `persistentEnclave-enclave-bundle-simulation`, etc